static/js/hotkey.js

```diff
@@ -198,6 +198,8 @@ function process_hotkey(e) {
         } else if ($("#subscription_overlay").is(":visible")) {
             subs.close();
             return true;
+        } else if ($('#markdown-help').hasClass('in')) {
+            return false;
```

Comment:
This should similarly check for search-help and the keyboard shortcuts; they use the same system.